src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/xlf/MicrosoftCodeQualityAnalyzersResources.pt-BR.xlf

```diff
@@ -269,7 +269,7 @@
       </trans-unit>
       <trans-unit id="UriParametersShouldNotBeStringsCodeFixTitle">
         <source>Add 'System.Uri' overloads</source>
-        <target state="new">Add 'System.Uri' overloads</target>
+        <target state="translated">Adicionar sobrecarga 'System.Uri'.</target>
```

Comment:
@mavasani Is it okay for translated code fix titles to end in period?

Reply:
@cristianosuzuki77 Can you verify that this is being translated as a sentence fragment (at minimum in a manner consistent with other titles)